addons/base_vat/tests/test_validate_ruc.py

```diff
@@ -148,6 +148,19 @@ def test_vat_vn(self):
         with self.assertRaisesRegex(ValidationError, msg):
             test_partner.write({'vat': '0123457890-11134'})
 
+    def test_vat_do(self):
+        test_partner = self.env["res.partner"].create({"name": "DO Company", "country_id": self.env.ref("base.do").id})
+        # Valid do vat
+        test_partner.write({"vat": "152-0000706-8"})
```

Comment:
Can you test that both the 9-digit and 11-digit ones pass? Just for completeness.